subsys/logging/log_core.c

```diff
@@ -393,3 +416,23 @@ u32_t log_filter_get(struct log_backend const *const backend,
 		return log_compiled_level_get(src_id);
 	}
 }
+
+#ifdef CONFIG_LOG_PROCESS_THREAD
+
+#define STACKSIZE 768
+
+static void log_process_thread(void *dummy1, void *dummy2, void *dummy3)
+{
+	log_init(k_current_get());
+
+	while (1) {
+		if (log_process(false) == false) {
+			k_sleep(CONFIG_LOG_PROCESS_THREAD_SLEEP_MS);
+		}
+	}
+}
+
+K_THREAD_DEFINE(log_process_thread_id, STACKSIZE, log_process_thread,
```

Comment:
call this 'log_process_thread', then it appears as such when we list threads by name (using stringify)